Parsers/Quote of the Day.js

```diff
@@ -0,0 +1,40 @@
+/*
+activation_example: !quote
+regex: !quote
+flags: gmi
+order: 100
+stop_processing: true
+active: true
+*/
+
+// GlideCacheManager to store quote for the day
+var cache = GlideCacheManager.get().getCache('daily_quote');
```

Comment:
GlideCacheManager is undocumented. Consider how you could use a solution like [ScopedCacheManager](https://developer.servicenow.com/dev.do#!/reference/api/xanadu/server/sn_scoped_cache-namespace/ScopedCacheManagerAPI), and additionally how you ensure you clean up after time has passed on a cache.